releases/v21.1.8.md

```diff
@@ -6,6 +6,11 @@ summary: Additions and changes in CockroachDB version v21.1.8 since version v21.
 
 ## August 30, 2021
 
+{{site.data.alerts.callout_danger}}
+This patch release has been withdrawn due to [this technical advisory](../advisories/a69874.html).
+{{site.data.alerts.end}}
+
+
```

Comment:
Should we comment out everything below this warning? @ianjevans, thoughts?

Reply:
My two cents would be to keep it -- v21.1.8 is already out in the wild, so users should have access to have release notes.

Reply:
Ok, that's fine.